frontend/src/metabase-types/api/dashboard.ts

```diff
@@ -72,3 +74,19 @@ export type DashCardDataMap = Record<
   DashCardId,
   Record<CardId, Dataset | undefined>
 >;
+
+export type LinkEntity = {
+  id: number;
+  db_id?: number;
+  database_id?: number;
```

Comment:
`DatabaseId`?

Reply:
to maintain compatibility with the `SearchResults` component this should stay `database_id`